src/include/reconcile.h

```diff
@@ -143,6 +143,27 @@ struct __wt_reconcile {
     size_t space_avail;     /* Remaining space in this chunk */
     size_t min_space_avail; /* Remaining space in this chunk to put a minimum size boundary */
 
+    /*
+     * Counters tracking how much time information is included in reconciliation for each page that
+     * is written to disk. The number of entries on a page is limited to a 32 bit number so these
+     * counters can be too.
+     */
+    uint32_t count_durable_start_ts;
```

Comment:
Is it worth moving these together with the stats added in  WT-6080? i.e.  bool rec_page_cell_with_ts;

Reply:
Ah - yup. There is a lot of overlap. I'll touch base with Haseeb. Thanks for the heads up.